src/config/2021.json

```diff
@@ -654,6 +712,14 @@
       "github": "kevinfarrugia",
       "twitter": "imkevdev"
     },
+    "kachiden": {
+      "name": "Katriel Paige",
```

Comment:
@kachiden FYI I've added your name but since it's not on your GitHub profile let me know if you'd rather go by something else